src/lib.rs

```diff
@@ -470,9 +470,12 @@ pub trait Labeller<'a,N,E> {
     }
 
     /// Maps `e` to arrow style that will be used on the end of an edge.
-    /// Defaults to normal.
+    /// Defaults to normal or none, depending on `kind()`.
     fn edge_end_arrow(&'a self, _e: &E) -> Arrow {
```

Comment:
I tried this first.

Drawbacks are that's it's a minor breaking change, since code that hardcodes in Arrow::normal() may start emitting explicit arrowhead= attributes. It would change the emitted dot files but not their rendering.

Another drawback is that the default cannot render to a dot string on its own.